From ebb3954562f9947a736af0eba77cc00d9d32a799 Mon Sep 17 00:00:00 2001
From: Juan Quintela <quintela@redhat.com>
Date: Mon, 5 Mar 2012 17:21:49 +0100
Subject: [PATCH 03/11] ram: use correct milliseconds amount

RH-Author: Juan Quintela <quintela@redhat.com>
Message-id: <d9b766b3491e47dd5b0fcb5d1ac10792263576fe.1330967827.git.quintela@redhat.com>
Patchwork-id: 37963
O-Subject: [PATCH qemu-kvm RHEL-6.3 1/3] ram: use correct milliseconds amount
Bugzilla: 752138
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>

From: Juan Quintela <quintela@trasno.org>

We were testing for 50.000 seconds instead of 50ms.

Signed-off-by: Juan Quintela <quintela@redhat.com>
---
 vl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 vl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vl.c b/vl.c
index b3275b0..fbf01ef 100644
--- a/vl.c
+++ b/vl.c
@@ -2985,7 +2985,7 @@ static int ram_save_live(Monitor *mon, QEMUFile *f, int stage, void *opaque)
        */
         if ((i & 63) == 0) {
             uint64_t t1 = (get_clock() - t0) / 1000000;
-            if (t1 > 50000000) { /* 50 ms */
+            if (t1 > 50) { /* 50 ms */
                break;
             }
         }
-- 
1.7.7.6

